crates/nu-command/src/filesystem/ls.rs

```diff
@@ -491,7 +491,10 @@ pub(crate) fn dir_entry_dict(
                         span,
                     });
                 } else {
-                    vals.push(Value::nothing(span))
+                    vals.push(Value::Int {
```

Comment:
Open question if `Int` or `String` is better here. String would keep the type of the column consistent but would be indistinguishable from a found username or groupname.

Reply:
I thought about it as well, but in the end I thought `Int` was better. It's not inconceivable for a username or groupname to be a number, and in those cases the user should have some way to distinguish the two I feel.

Reply:
https://unix.stackexchange.com/questions/287077/why-cant-linux-usernames-begin-with-numbers Welp, some people really want to see the world burn...

I am OK with your decision. Hope that the potential downstream commands are fine with mixed types.